composer.json

```diff
@@ -33,9 +33,11 @@
         "knplabs/gaufrette": "^0.8 || ^0.9",
         "kriswallsmith/buzz": "^0.15 || ^0.16",
         "psr/log": "^1.0",
-        "sonata-project/core-bundle": "^3.14",
+        "sonata-project/block-bundle": "^3.17",
```

Comment:
not sure about this one... this dependency was optional, and not it is required. Not sure we have to link the metadatainterface to the block bundle

Reply:
https://github.com/sonata-project/SonataBlockBundle/pull/506
https://github.com/sonata-project/SonataMediaBundle/commit/206b4cb26065fbd887bebc40a791aef6b56b15a9
https://github.com/sonata-project/SonataMediaBundle/commit/5cda60ead57473db7f31db1e3bcab3698f427c11

@core23 
Mybe you remeber something? Should we keep it?  

For me it is another reason to discouse about: https://github.com/sonata-project/dev-kit/issues/658. 

This will be awosome feature to create auto generating things like media put to block and display in site or site composer  (PageBundle) based on metadatas with some context.

Reply:
You're right. The dependency should be optional. There is no need for the block bundle, see: https://github.com/sonata-project/SonataMediaBundle/pull/1744#discussion_r437643302